### DISCUSSION TITLE
关于类SQL指令的问题

### DISCUSSION THREAD

**Discussion by NishinoAsuka0** [liusy58/NJU_AdvancedProgramming_SP22#127] [Q&A] [answered]
我想请问类SQL指令究竟是什么要求呢？是要我自己手动输入类SQL指令的格式到一个函数里面然后体现出SQL指令对应的功能吗？

**Comment by binbin00** [comment, marked as answer]
你可以这样想，你用的一个数据库存储数据，这个数据库需要指令才能对数据进行操作，所以你首先要把角色的操作先转化为指令。然后，因为这个数据库也是你实现的，所以你需要再解析指令并对数据进行对应的操作。（这个解释有点生硬，希望你能理解，我们对这块的要求不高，原本是希望有 用户能输入指令 的功能的，但这对大家来说很烦。）
